node/pkg/watchers/ibc/watcher.go

```diff
@@ -63,7 +63,13 @@ func setFeatures(f string) {
 
 var (
 	// Chains defines the list of chains to be monitored by IBC. Add new chains here as necessary.
-	Chains = []vaa.ChainID{vaa.ChainIDSei}
+	Chains = []vaa.ChainID{
```

Comment:
Just want to clarify that with this change, these chains will be able to emit messages as soon as the channel is allow-listed with governance - is that the idea?

Reply:
Yes. When the watcher receives a message for a channel it does not know about, it queries the ibc-receiver contract for the channel to chain mapping. If for some reason, the contract doesn't have the channel to chain mapping (meaning governance hasn't happened yet, but if governance hasn't happened yet, the contract shouldn't publish the message), the watcher generates an error message like "received message from unknown IBC channel" and drops the message.